config.go

```diff
@@ -0,0 +1,10 @@
+package main
+
+// Config holds the step inputs
+type Config struct {
+	TestName       string `env:"test_name,required"`
+	SearchPath     string `env:"search_path,required"`
+	TestResultsDir string `env:"bitrise_test_result_dir,required"`
```

Comment:
there is a `dir` struct field tag, which also validates if the given dir exists or not